daemonset/kubeadm-kuberouter-all-features-hostport.yaml

```diff
@@ -0,0 +1,184 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-router-cfg
+  namespace: kube-system
+  labels:
+    tier: node
+    k8s-app: kube-router
+data:
+  cni-conf.json: |
+    {
+       "cniVersion":"0.3.0",
```

Comment:
Isn't `0.6.0` the most recent stable version? Why 0.3.0?

Reply:
This is CNI spec version https://github.com/containernetworking/cni/blob/master/SPEC.md Plug-in's vesioning is different. I got to know about it for this PR :)

Reply:
TIL